### PR TITLE
ChemAxon.jchemdotnetapi version 21.15.2

### PR DESCRIPTION
**Pull request by oxygen-dioxide**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/78833)

### CODE REVIEW

File: manifests/c/ChemAxon/jchemdotnetapi/21.15.2/ChemAxon.jchemdotnetapi.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: ChemAxon.jchemdotnetapi
+PackageVersion: 21.15.2
```
Comment (ItzLevvie) [inline manifests/c/ChemAxon/jchemdotnetapi/21.15.2/ChemAxon.jchemdotnetapi.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 21.15.2.9
```
```
Enter a pull request number: 78833

Manifest validation succeeded.
Found JChem .NET API [ChemAxon.jchemdotnetapi] Version 21.15.2
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://dl.chemaxon.com/jchem-net-api/21.15.2/JChem_NET_API_21.15.2.x64.msi
  ██████████████████████████████   144 MB /  144 MB
Successfully verified installer hash
Starting package install...
Successfully installed

Name          : JChem .NET API 21.15.2.9
Publisher     : ChemAxon
Version       : 21.15.2.9
ProductCode   : {df2ce283-fc56-4509-a1f0-8cc56a46838a}
Registry Path : Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\{df2ce283-fc56-4509-a1f0-8cc56a46838a}
Uninstall     : winget uninstall --id "{df2ce283-fc56-4509-a1f0-8cc56a46838a}"

Press any key to continue . . .
```

File: manifests/c/ChemAxon/jchemdotnetapi/21.15.2/ChemAxon.jchemdotnetapi.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: ChemAxon.jchemdotnetapi
+PackageVersion: 21.15.2
```
Comment (ItzLevvie) [inline manifests/c/ChemAxon/jchemdotnetapi/21.15.2/ChemAxon.jchemdotnetapi.yaml:5]:
```suggestion
PackageVersion: 21.15.2.9
```
```
Enter a pull request number: 78833

Manifest validation succeeded.
Found JChem .NET API [ChemAxon.jchemdotnetapi] Version 21.15.2
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://dl.chemaxon.com/jchem-net-api/21.15.2/JChem_NET_API_21.15.2.x64.msi
  ██████████████████████████████   144 MB /  144 MB
Successfully verified installer hash
Starting package install...
Successfully installed

Name          : JChem .NET API 21.15.2.9
Publisher     : ChemAxon
Version       : 21.15.2.9
ProductCode   : {df2ce283-fc56-4509-a1f0-8cc56a46838a}
Registry Path : Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\{df2ce283-fc56-4509-a1f0-8cc56a46838a}
Uninstall     : winget uninstall --id "{df2ce283-fc56-4509-a1f0-8cc56a46838a}"

Press any key to continue . . .
```

File: manifests/c/ChemAxon/jchemdotnetapi/21.15.2/ChemAxon.jchemdotnetapi.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: ChemAxon.jchemdotnetapi
+PackageVersion: 21.15.2
```
Comment (ItzLevvie) [inline manifests/c/ChemAxon/jchemdotnetapi/21.15.2/ChemAxon.jchemdotnetapi.installer.yaml:5]:
```suggestion
PackageVersion: 21.15.2.9
```
```
Enter a pull request number: 78833

Manifest validation succeeded.
Found JChem .NET API [ChemAxon.jchemdotnetapi] Version 21.15.2
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://dl.chemaxon.com/jchem-net-api/21.15.2/JChem_NET_API_21.15.2.x64.msi
  ██████████████████████████████   144 MB /  144 MB
Successfully verified installer hash
Starting package install...
Successfully installed

Name          : JChem .NET API 21.15.2.9
Publisher     : ChemAxon
Version       : 21.15.2.9
ProductCode   : {df2ce283-fc56-4509-a1f0-8cc56a46838a}
Registry Path : Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\{df2ce283-fc56-4509-a1f0-8cc56a46838a}
Uninstall     : winget uninstall --id "{df2ce283-fc56-4509-a1f0-8cc56a46838a}"

Press any key to continue . . .
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
See https://github.com/microsoft/winget-pkgs/pull/78833#pullrequestreview-1103017798

**Review by ItzLevvie — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**